src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -166,6 +166,11 @@ public partial class DynamoModel : IDynamoModel, IDisposable, IEngineControllerM
         /// </summary>
         public static bool EnableMigrationLogging { get; set; }
 
+        /// <summary>
+        /// The Autodesk CrashReport tool location on disk (directory that contains the "senddmp.exe")
+        /// </summary>
+        public static string CrashToolLocation;
```

Comment:
so how would `DynamoRevit` set this?

Reply:
I will add a public setter and getter
Any other suggestions that may be a better fit for the API ?
Maybe adding it to Preferences or to expand the StartConfiguration ?

Reply:
I think it makes sense in the start configuration, though I know that is a bit of a pain - preferences does not make as much a sense to me as it's not a user preference but an integration one.